test/acceptance/get_with_invalid_filter_and_include_test.exs

```diff
@@ -0,0 +1,248 @@
+# SPDX-FileCopyrightText: 2019 ash_json_api contributors <https://github.com/ash-project/ash_json_api/graphs.contributors>
+#
+# SPDX-License-Identifier: MIT
+
+defmodule Test.Acceptance.GetWithInvalidFilterAndIncludeTest do
```

Comment:
This testing looks good, but I'd rather that we added it to an existing test if possible. I think there should be like a `get_test.exs` somewhere else for example.

Reply:
I have added tests to get_related_tests.exs.